packages/gatsby/package.json

```diff
@@ -48,7 +48,7 @@
     "domready": "^1.0.8",
     "dotenv": "^4.0.0",
     "eslint": "^4.19.1",
```

Comment:
Should we take this opportunity to bump eslint to 5.0.0? Worth investigating to see if our lint setup still passes upgrading both?